app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -212,5 +231,8 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
             "ara", // Arabic
             "jpn" // Japanese
         )
+
+        private val PREFERENCE_KEY = "LEANPLUM_PREFERENCES"
```

Comment:
super nit picky: `PREFERENCE_KEY` is technically not a "key" but the file name, I think?